app/src/main/java/com/infomaniak/swisstransfer/ui/screen/newtransfer/ImportationFilesManager.kt

```diff
@@ -92,12 +92,11 @@ class ImportationFilesManager @Inject constructor(
     suspend fun continuouslyCopyPickedFilesToLocalStorage() {
         filesToImportChannel.consume { fileToImport ->
             SentryLog.i(TAG, "Importing ${fileToImport.uri}")
-            val copiedFile = copyUriDataLocally(fileToImport.uri, fileToImport.fileName)
-
-            if (copiedFile == null) {
-                reportFailedImportation(fileToImport)
-                return@consume
-            }
+            val copiedFile = Dispatchers.IO {
```

Comment:
What's the reason that motivates us to use Dispatchers.IO here, I saw you mention that we used dispatchers in a wrong manner sometimes and that it should only be used for "blocking code parts". Yet this isn't a suspend method, I'd be curious to learn more

Reply:
Discussed offline